46_113_path_sum_II_final (1).py

```diff
@@ -0,0 +1,163 @@
+# Pallavi Sapale
+

```

Comment:
Try to code the solution as taught in class. It might be that some functions are not accepted in Leetcode but works if other functions are used. Just give it a try with different approach. Let me know if it doesn't work